eslint.config.cjs

```diff
@@ -65,6 +71,15 @@ module.exports = tseslint.config(
             "@typescript-eslint/explicit-function-return-type": "off",
             "@typescript-eslint/no-non-null-assertion": "off",
             "@typescript-eslint/no-explicit-any": "off",
+            "@typescript-eslint/restrict-template-expressions": "off", // too jumpy
+            "@typescript-eslint/no-misused-promises": "off", // too jumpy
+            "@typescript-eslint/no-floating-promises": "off", // jumpy; would be nice to turn on, but we have a lot of these
+
+            // This codebase interacts with `any` a lot, so for now, all no-unsafe-* we violate are currently disabled
```

Comment:
A good compromise is to allow code to explicitly use `foo: any` to opt-in to "any". I think that's a tsconfig option, IIRC.

Reply:
I think we do, but maybe not. I'll check.

This basically prevents `any` from escaping your current scope, which I'm not sure that casting will satisfy.